packages/client/src/i18n/nl.ts

```diff
@@ -91,6 +91,10 @@ export default {
           "Voer het adres in waar u de dakkapel wilt gaan plaatsen.",
       },
     },
+    notFoundPage: {
+      heading: "Helaas",
+      paragraph: "Deze pagina is niet gevonden",
```

Comment:
Please add a dot/period after the sentence "Deze pagina is niet gevonden" and replace the original text in NotFoundPage/tsx

Also, heading and paragraph should be swapped, since this is the formatting:
```
<Heading>Deze pagina is niet gevonden.</Heading>
<Paragraph>Helaas</Paragraph>
```